modules/git/git.go

```diff
@@ -20,10 +21,8 @@ import (
 )
 
 var (
-	// Prefix the log prefix
-	Prefix = "[git-module] "
```

Comment:
Was that simply unused?

Reply:
Yes, not used anymore. Otherwise there would be an error when compile.